<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AT_manager_client</class>
 <widget class="QMainWindow" name="AT_manager_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>atmc-mini_1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="mainBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Основные операции</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Работа со скриптом</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pbTranslate">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Транслировать скрипт</string>
             </property>
             <property name="iconSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbStep">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Выполнить шаг по коду</string>
             </property>
             <property name="iconSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbRun">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(174, 185, 255);</string>
             </property>
             <property name="text">
              <string>Запустить скрипт</string>
             </property>
             <property name="iconSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>pbRun</zorder>
          <zorder>verticalSpacer</zorder>
          <zorder>pbTranslate</zorder>
          <zorder>pbStep</zorder>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Загрузка\сохранение скриптов</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pbSaveScript">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Сохранить фаил</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbOpenScipt">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Открыть фаил</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>293</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Скрипт</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="font">
            <font>
             <family>System</family>
             <pointsize>10</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(9, 7, 7);
color: rgb(22, 150, 8);
font: 75 10pt &quot;System&quot;;</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Отладочный код</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Lucida Console&quot;;
background-color: rgb(243, 231, 255);
color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_26">
        <attribute name="title">
         <string>Отчет</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSaveScript_2">
           <property name="text">
            <string>Сохранить в html</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QToolBox" name="toolBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>6</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>360</width>
          <height>484</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Порты ввода\вывода</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QScrollArea" name="sa_UART1_2">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_19">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>323</width>
              <height>447</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout_28">
             <item row="0" column="0">
              <widget class="QGroupBox" name="gb_Port_value">
               <property name="title">
                <string>Значения</string>
               </property>
               <layout class="QFormLayout" name="formLayout_8">
                <item row="0" column="0">
                 <widget class="QLabel" name="txt_PortA_value">
                  <property name="text">
                   <string>Порт A :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_PortA_value">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="txt_PortB_value">
                  <property name="text">
                   <string>Порт B :</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="lb_PortB_value">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="txt_PortC_value">
                  <property name="text">
                   <string>Порт C :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="lb_PortC_value">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="txt_PortD_value">
                  <property name="text">
                   <string>Порт D :</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="lb_PortD_value">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="gb_Cfg">
               <property name="title">
                <string>Конфигурация</string>
               </property>
               <layout class="QFormLayout" name="formLayout_9">
                <item row="0" column="0">
                 <widget class="QLabel" name="txt_PortA_ddr">
                  <property name="text">
                   <string>DDRA</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_PortA_ddr">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="txt_PortB_ddr">
                  <property name="text">
                   <string>DDRB: </string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="lb_PortA_ddr_2">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="txt_PortC_ddr">
                  <property name="text">
                   <string>DDRC:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="lb_PortA_ddr_3">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="txt_PortD_ddr">
                  <property name="text">
                   <string>DDRD:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="lb_PortA_ddr_4">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QGroupBox" name="gb_Stats">
               <property name="title">
                <string>Статистика количества обращений к </string>
               </property>
               <layout class="QFormLayout" name="formLayout_24">
                <item row="0" column="0">
                 <widget class="QLabel" name="txt_PortA_stats">
                  <property name="text">
                   <string>Порту A:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_PortA_stats">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="txt_PortB_stats">
                  <property name="text">
                   <string>Порту B:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="lb_PortB_stats">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="txt_PortC_stats">
                  <property name="text">
                   <string>Порту C:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="lb_PortC_stats">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="txt_PortD_stats">
                  <property name="text">
                   <string>Порту D:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="lb_PortD_stats">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>unknown</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>360</width>
          <height>484</height>
         </rect>
        </property>
        <attribute name="label">
         <string>ДРУГИЕ РЕГИСТРЫ</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="txt_UReg1">
           <property name="text">
            <string>Пользовательский регистр 1:   </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lb_UReg1">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="txt_UReg2">
           <property name="text">
            <string>Пользовательский регистр 2:   </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lb_UReg2">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="txt_UReg3">
           <property name="text">
            <string>Пользовательский регистр 3:   </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lb_UReg3">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="txt_UReg4">
           <property name="text">
            <string>Пользовательский регистр 4:   </string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lb_UReg4">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="txt_UReg5">
           <property name="text">
            <string>Пользовательский регистр 5:   </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lb_UReg5">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="lb_UReg6">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="txt_UReg6">
           <property name="text">
            <string>Пользовательский регистр 6:   </string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="txt_UReg7">
           <property name="text">
            <string>Пользовательский регистр 7:   </string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="lb_UReg7">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="txt_UReg8">
           <property name="text">
            <string>Пользовательский регистр 8:   </string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="txt_Freg">
           <property name="text">
            <string>Флаговый регистр:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="lb_UReg8">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="lbFReg">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>pbRun</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
